io/src/main/java/org/commonjava/maven/ext/io/SettingsIO.java

```diff
@@ -57,7 +58,7 @@ public void write( Settings settings, File settingsFile )
     {
         try
         {
-            PrintWriter printWriter = new PrintWriter( settingsFile, settings.getModelEncoding() );
+            PrintWriter printWriter = new PrintWriter( settingsFile, StandardCharsets.UTF_8.toString() );
```

Comment:
Why change here? You mentioned partial models in test as a reason, but this is not a test case. This file could have a different encoding set by the user. Recommend:

```
settings.getModelEncoding() != null ? settings.getModelEncoding() : StandardCarsets_UTF_8.name();
```

Reply:
Agreed. ( looked at 4.3.3 in https://www.w3.org/TR/xml/ )